hbase-archetypes/hbase-archetype-builder/pom.xml

```diff
@@ -179,6 +179,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>xml-maven-plugin</artifactId>
+        <version>${xml.maven.version}</version>
```

Comment:
Hi @Apache9 the plugin is still there. Do we want to modify commit message/JIRA  title?

Reply:
Changed to "Remove the xml-maven-plugin in root pom on branch-3"